core/icons/src/vue/c-icon.vue

```diff
@@ -3,17 +3,21 @@ import "bootstrap-icons/font/bootstrap-icons.css";
 
 defineProps<{
   name: string;
+  size: string;
```

Comment:
I suggest making this size optional, as afaics many uses of `<c-icon>` on templates does not define it.
Also, since it can be undefined, this needs to be taken into account.
When undefined, the classes generated on the span are `cr-icon bi-NAME undefined`, where we would like `cr-icon bi-NAME`.
```suggestion
  size?: string;
```

Reply:
Ah ok, got it.